lib/matplotlib/patches.py

```diff
@@ -120,17 +119,52 @@ def get_verts(self):
             return polygons[0]
         return []
 
+    def set_picker(self, p):
+        # docstring inherited
+        # After deprecation, the whole method can be deleted and inherited.
+        if isinstance(p, Number) and not isinstance(p, bool):
+            cbook.warn_deprecated(
+                "3.4", message="Setting the collections's pick radius via "
+                "set_picker is deprecated since %(since)s and will be removed "
+                "%(removal)s; use set_pickradius instead.")
+            self.set_pickradius(p)
+        self._picker = p
+
+    def set_pickradius(self, d):
```

Comment:
Did `Patch` not have `set_pickradius` before?  I'm concerned about deprecating this and adding the replacement API in the same version.